src/main/kotlin/com/openlattice/collections/EntitySetCollection.kt

```diff
@@ -0,0 +1,64 @@
+package com.openlattice.collections
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.IdConstants
+import com.openlattice.authorization.securable.AbstractSecurableObject
+import com.openlattice.authorization.securable.SecurableObjectType
+import com.openlattice.client.serialization.SerializationConstants
+import java.util.*
+
+
+class EntitySetCollection(
+        id: Optional<UUID>,
+        var name: String,
+        title: String,
+        description: Optional<String>,
+        val entityTypeCollectionId: UUID,
+        var template: MutableMap<UUID, UUID>,
+        var contacts: Set<String>,
+        var organizationId: UUID
+) : AbstractSecurableObject(id, title, description) {
+
+    /**
+     * Creates an entity set collection with provided parameters and will automatically generate a UUID if not provided.
+     *
+     * @param id An optional UUID for the entity set collection.
+     * @param name The unique name of the entity set collection.
+     * @param title The friendly name for the entity set collection.
+     * @param description A description of the entity set collection.
+     * @param entityTypeCollectionId The id of the entity type collection that this entity set collection maps to.
+     * @param template A mapping from collection template type ids to entity set ids.
+     * @param contacts A set of contact methods for the owners of this entity set collection.
+     * @param organizationId The id of the organization that this entity set collection belongs to.
+     */
+    @JsonCreator
+    constructor(
+            @JsonProperty(SerializationConstants.ID_FIELD) id: Optional<UUID>,
```

Comment:
Any chance you can update orgaizationId to have `IdConstants.GLOBAL_ORGANIZATION_ID.id` as a default value?

Reply:
ooh yes good call